### ISSUE TITLE
Keyboard navigation does not work as expected

### ISSUE THREAD

**Issue by julienfitz** [rust-lang/www.rust-lang.org#37]
If you try to navigate using the keyboard, you should be able to tab through the elements until you reach a navigation menu item, then you should be able to hit the space bar to expand that menu option. However, hitting the space bar does not seem to work here (it just scrolls halfway down the page on Chrome) and on Firefox tab navigation does not seem to work at all (hitting tab doesn't seem to bring you to any of the navigation items).
